accountsplus/templates/accounts/locked_out.html

```diff
@@ -0,0 +1,31 @@
+{% load bootstrap3 %}
+
+{% block content %}
+
+    <div class="col-lg-6 col-lg-offset-3 col-md-6 col-md-offset-3 locked-out-container">
+        <div class="login-message">
+            Your login is blocked due to multiple failing login attempts. Solve the following captcha to get unblocked!
```

Comment:
@nasief I suggest to put this string in `blocktrans` so if someone uses this library, he/she can translate these values